lib/index/response_validator.rb

```diff
@@ -0,0 +1,61 @@
+module Index
+  class ResponseValidator
+    class NotFound < StandardError; end
+    class ElasticsearchError < StandardError; end
+
+    def self.valid!(response)
```

Comment:
How come we have both `valid!` and `valid?`? Shouldn't one call the other?

If they are meant to be index specific the name doesn't make that clear enough.

Reply:
this is a temporary measure as i want both in the same spot, but still thinking about the best way to combine functionality.

Reply:
I plan this leave this as is, maybe try to refactor in the new year if i have time.

Reply:
can you give them different names for now then? I think it's easy to confuse the two at the moment.